semantic-conventions/syntax.md

```diff
@@ -333,7 +339,38 @@ fields are present in the current attribute definition, they override the inheri
 #### Type
 
 An attribute type can either be a string, int, double, boolean, array of strings, array of int, array of double,
-array of booleans, or an enumeration. If it is an enumeration, additional fields are required:
+array of booleans, a template type or an enumeration. 
+
+##### Template type
+
+A template type attribute represents a _key-value set_ of attributes with a common attribute name prefix. The syntax for defining template type attributes is the following:
```

Comment:
```suggestion
A template type attribute represents a _map/dictionary_ of attributes with a common attribute name prefix. The syntax for defining template type attributes is the following:
```

key-value set is an unusual term, and I think it would imply that you can have the same key multiple times, as long as the values are different.